### ISSUE TITLE
Unable to scroll history in conversations without using a mouse (i.e. with PageUp/PageDown)

### ISSUE THREAD

**Issue by max-baz** [wireapp/wire-webapp#549]
Open an Otto the Bot chat.
Exchange a few messages so that the history does not fit one screen and requires scrolling.
Try to scroll the history without using a mouse (natural thought: PageUp / PageDown).

Nothing happens, the page does not scroll.

----

I had a quick look at the code, it doesn't seem to be a problem with javascript - the keydown events are bubbled and not cancelled. I believe the problem is with css, thought I don't have a proof. I found elements with classes that are called `antiscroll-inner` and `antiscroll-wrap`, but simple removal of these classes does not make scrolling work.

**Comment by bennycode** [comment]
We are supporting navigating with Page Up / Page Down keys now in our webapp.
